src/lib/components/auth/FrostedCard.svelte

```
@@ -0,0 +1,14 @@
+  export let className: string = "";
+  
+  // Function to handle mouse down event, if needed
+  export let onMouseDown: ((e: MouseEvent) => void) | undefined = undefined;
+</script>
+
+<div 
+  class="mx-auto w-full max-w-md overflow-hidden rounded-lg bg-white/30 backdrop-blur-xl backdrop-filter shadow-lg border border-white/20 {className}"
+  on:mousedown={onMouseDown}
+>
+  <slot></slot>
+</div>
```

ESLint

[error] A11y: <div> with mousedown handler must have an ARIA role(a11y-no-static-element-interactions)

(svelte/valid-compile)